src/main/java/org/javacord/bot/commands/DocsCommand.java

```diff
@@ -70,10 +90,22 @@ public void onCommand(TextChannel channel, String[] args) throws IOException {
      * @param embed        The embed to populate.
      * @param searchString A search string.
      */
-    private void populateMethods(DiscordApi api, EmbedBuilder embed, String searchString) {
-        Map<String, List<JavadocMethod>> methods = new JavadocParser(api, JavadocParser.getLatestJavaDocs(api).join())
-                .getMethods().join().stream()
+    private void populateMethods(DiscordApi api, EmbedBuilder embed, String searchString, boolean includeAll) {
+        CompletableFuture<Set<JavadocMethod>> apiMethods = JavadocParser.getLatestJavaDocs(api)
+                .thenApply(urlString -> new JavadocParser(api, urlString))
+                .thenCompose(JavadocParser::getMethods);
+        CompletableFuture<Set<JavadocMethod>> coreMethods = (includeAll)
+                ? JavadocParser.getLatestCoreJavaDocs(api)
+                .thenApply(urlString -> new JavadocParser(api, urlString))
+                .thenCompose(JavadocParser::getMethods)
+                : CompletableFuture.completedFuture(Collections.emptySet());
+
+        Map<String, List<JavadocMethod>> methods = apiMethods.thenCombine(coreMethods, this::unionOf).join().stream()
                 .filter(method -> method.getFullName().toLowerCase().contains(searchString.toLowerCase()))
+                .filter(method -> {
+                    String packageName = method.getPackageName();
+                    return includeAll || !(packageName.endsWith(".internal") || !packageName.contains(".internal."));
```

Comment:
Your boolean logic is still broken, now you don't show anything for `!includeAll`, except it is in a "subpackage" of `internal`.

It should probably have been like
```suggestion
                    return includeAll || !(packageName.endsWith(".internal") || packageName.contains(".internal."));
```